src/App.js

```diff
@@ -1,23 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import ProductPage from "./components/ProductsPage";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <ProductPage></ProductPage>
```

Comment:
When its only one component and dont recive any children you can close the tag in the same component
```suggestion
      <ProductPage/>
```